### ISSUE TITLE
Default link opening feature for spotify/yt music

### ISSUE THREAD

**Issue by sPKR-ai** [Sangwan5688/BlackHole#259]
I use blackhole. But at present most people don't. So they share links from other apps. It would be nice if spotify/yt music links could default to opening in this app. Also it would be nice if we had an option to share spotify/yt music links of the song directly from blackhole.

NB: Jiosaavn already works well with blackhole.

**Comment by Sangwan5688** [comment]
Cool, would be a nice addition

**Comment by luphoria** [comment]
> I use blackhole. But at present most people don't. So they share links from other apps. It would be nice if spotify/yt music links could default to opening in this app. Also it would be nice if we had an option to share spotify/yt music links of the song directly from blackhole.
> 
> NB: Jiosaavn already works well with blackhole.

yes!! this is something that is blatantly missing from tons of these services
